deploy/crds/aws_v1alpha1_account_crd.yaml

```diff
@@ -52,6 +52,8 @@ spec:
               type: string
             iamUserSecret:
               type: string
+            supportCaseID:
+              type: string
```

Comment:
@jharrington22 What do you mean here?

Reply:
@fahlmant moving the `supportCaseID` field to the Status section, instead of keeping it under Spec

Reply:
I mean the second part. ` it doesn't follow K8s patterns where spec should not change unless a CR is applied.`

Reply:
For more context: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status

```quote
The Kubernetes API also serves as the foundation for the declarative configuration schema for the system. In order to facilitate level-based operation and expression of declarative configuration, fields in the specification should have declarative rather than imperative names and semantics -- they represent the desired state, not actions intended to yield the desired state.```

It means we should revisit other misplaced fields in the future

Reply:
Makes sense. +1 to what James suggested

Reply:
I has been updated, could any of you re-check please?

Reply:
I see you added it to the status in the account_types.go, but the field is still showing in the spec in the crd.